vision_agent_tools/models/owlv2.py

```diff
@@ -247,10 +282,18 @@ def post_process_object_detection_with_nms(
             boxes = boxes * scale_fct[:, None, :]
 
         results = []
-        for s, l, b in zip(scores, labels, boxes):
-            score = s[s > threshold]
-            label = l[s > threshold]
-            box = b[s > threshold]
-            results.append({"scores": score, "labels": label, "boxes": box})
+        for score_array, label_array, box_array in zip(scores, labels, boxes):
+            high_score_mask = score_array > threshold
+            filtered_scores = score_array[high_score_mask]
```

Comment:
these changes are required by `ruff`, not sure how it got merged before, they are not related to the PR but otherwise I cannot commit at all.